src/app/ui-helper.js

```diff
@@ -125,35 +136,34 @@ module.exports = {
     var details = $('<div style="display: none;"/>')
       .append(this.tableRow('Solidity Interface', contract.solidity_interface));
 
-    if (contract.opcodes !== '') {
-      details.append(this.tableRow('Opcodes', contract.opcodes));
-    }
-
     var funHashes = '';
     for (var fun in contract.functionHashes) {
       funHashes += contract.functionHashes[fun] + ' ' + fun + '\n';
     }
-    details.append($('<span class="col1">Functions</span>'));
-    details.append($('<pre/>').text(funHashes));
+    details.append(this.preRow('Functions', funHashes));
 
     var gasEstimates = this.formatGasEstimates(contract.gasEstimates);
     if (gasEstimates) {
-      details.append($('<span class="col1">Gas Estimates</span>'));
-      details.append($('<pre/>').text(gasEstimates));
+      details.append(this.preRow('Gas Estimates', gasEstimates));
     }
 
     if (contract.runtimeBytecode && contract.runtimeBytecode.length > 0) {
       details.append(this.tableRow('Runtime Bytecode', contract.runtimeBytecode));
     }
 
+    if (contract.opcodes !== '') {
```

Comment:
Can you change it to `!== undefined && !== ''` in case we remove from abstract contracts it in the future?